code/modules/mob/living/carbon/species.dm

```diff
@@ -250,41 +251,34 @@
 				H << "\red You feel your face freezing and an icicle forming in your lungs!"
 		else if(breath.temperature > heat_level_1)
 			if(prob(20))
-				if(H.dna.mutantrace == "slime")
-					H << "\red You feel supercharged by the extreme heat!"
-				else
-					H << "\red You feel your face burning and a searing heat in your lungs!"
-
-		if(H.dna.mutantrace == "slime")
```

Comment:
Functionality loss, should probably be updated to species check. Also doesn't really make sense if it tells slime people that their 'lungs' are burning.